internal/iamauthenticator/install.go

```diff
@@ -43,3 +51,13 @@ func Install(ctx context.Context, tracker *tracker.Tracker, iamAuthSrc IAMAuthen
 func Uninstall() error {
 	return os.RemoveAll(IAMAuthenticatorBinPath)
 }
+
+func Upgrade(ctx context.Context, src IAMAuthenticatorSource, log *zap.Logger) error {
```

Comment:
same here about those helper to download files, maybe we can use those to test these